documentation/source/user_section/tutorials/registration_to_template.rst

```diff
@@ -0,0 +1,375 @@
+.. _registration-to-template:
+
+Registration to template
+########################
+
+This tutorial demonstrates how to use SCT's command-line scripts to register an anatomical MRI scan to the PAM50 Template, then use the results of this registration to perform basic quantitative analysis. We recommend that you read through the :ref:`pam50` page before starting this tutorial, as it provides an overview of the template included with SCT, as well as context for why it is used.
+
+.. warning::
+
+   This tutorial uses sample MRI images that must be retrieved beforehand. Please download and unzip `sct_course_london20.zip <https://osf.io/bze7v/?action=download>`_, then open up the unzipped folder in your terminal and verify its contents using ``ls``.
+
+   .. code:: sh
+
+      ls
+      # Output:
+      # multi_subject single_subject
+
+   We will be using images from the ``single_subject/data`` directory, so navigate there and verify that it contains subdirectories for various MRI image contrasts using ``ls``.
+
+   .. code:: sh
+
+      cd single_subject/data
+      ls
+      # Output:
+      # dmri  fmri  LICENSE.txt  mt  README.txt  t1  t2  t2s
+
+   .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_image.png
+      :align: right
+      :figwidth: 8%
+
+      t2.nii.gz
+
+   We will start with T2-weighted images, so navigate to the ``t2`` directory and verify that it contains a T2-weighted anatomical image called ``t2.nii.gz``.
+
+   .. code:: sh
+
+      cd t2
+      ls
+      # Output:
+      # t2.nii.gz
+
+.. _segmentation-section:
+
+Step 1: Segmenting the spinal cord
+**********************************
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/registration-pipeline-1.png
+   :align: center
+   :figwidth: 75%
+
+   Steps performed by ``sct_deepseg_sc``.
+
+.. note::
+
+   The :ref:`spinalcord-segmentation` tutorial provides a more in-depth look at spinal cord segmentation. If you have already completed that tutorial, your ``t2/`` directory should contain a file called ``t2_seg.nii.gz``. If it does, you may skip this section and proceed with the :ref:`vert-labeling-section` section.
+
+Theory
+======
+
+First, we will run the ``sct_deepseg_sc`` command to segment the spinal cord from the anatomical image. The segmented spinal cord is a requirement for the vertebral/disc labeling stage that comes after.
```

Comment:
```suggestion
First, we will run the ``sct_deepseg_sc`` command to segment the spinal cord from the anatomical image. The segmented spinal cord is a requirement for the vertebral/disc labeling stage that comes after. It is also a requirement for registration to the template (because steps of the registration involves the segmentation) and for computing cord morphometrics, such as CSA.
```